### PR TITLE
#167897145 Setup babel

### PR DESCRIPTION
**Pull request by mthamayor**
#### What does this PR do?
Setup babel transpiler

#### Description of Task to be completed?
- Set up babel transpiler to enable the use of es6+ features
- Add start, development and build scripts
- Update README.md to reflect new changes

#### How should this be manually tested?
- CD into the root folder
- Run `npm run build` in your CLI. This will generate a **dist** folder that contains the transpiled project 
- You can run `npm run start` to start up the transpiled version of the application
- You can run `npm run dev` to start up the development server

#### Any background context you want to provide?
You need to have npm version `5.2+` or greater installed as the scripts are run with `npx` which requires the aforementioned npm version or higher

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/167897145

#### Screenshots (if appropriate)
None

#### Questions:
None

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  [
+    "@babel/env",
+    {
+      useBuiltIns: "usage",
+      corejs: 3,
+    },
+  ],
+];
+
+module.exports = { presets };
```
Comment (okabamac) [inline babel.config.js]: This looks great to me. Great job

**Review by sebaztyn — APPROVED** [review]
This is fine.

File: babel.config.js (hunk 1 above)
Comment (mthamayor) [reply babel.config.js]: Hello, Thank you for your feedback. Babel recommends babel.config.js for configuration. You can check this configuration page https://babeljs.io/docs/en/configuration

**Review by okabamac — APPROVED** [review]

**Review by Stylll — COMMENTED** [review]
Hey @mthamayor Good Job!. Please resolve the merge conflicts.

File: package.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
   "license": "MIT",
```
Comment (Stylll) [inline package.json:14]: hey @mthamayor, I noticed you removed these script commands you added before. Is there a reason why?

**Review by Stylll — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (mthamayor) [reply package.json:14]: Hello, thank you for bring my attention to this. It's a mistake on my end. I'll work on this as soon as possible.

File: babel.config.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+/* eslint linebreak-style: ["error", "windows"] */
```
Comment (Stylll) [inline babel.config.js]: Hey @mthamayor , I don't think this line is necessary here. You may want to remove it. I believe that is why the hound bot is giving you an error about linebreaks.

**Review by Stylll — CHANGES_REQUESTED** [review]
Great job @mthamayor . There's one more change required. Kindly look into it.

File: babel.config.js (hunk 3 above)
Comment (mthamayor) [reply babel.config.js]: Hello, I have worked on it as requested

**Review by Stylll — APPROVED** [review]

**Pull request merged**